bot/bot.py

```diff
@@ -77,28 +77,39 @@ async def remove_word(update: Update, context: ContextTypes.DEFAULT_TYPE):
 
 
 
-def get_random_quiz():
+def get_random_quiz(language=None):
     filename_json = 'words.json'
 
     # Load the JSON file
     with open(filename_json, 'r', encoding='utf-8') as f:
         word_list = json.load(f)
 
+    # Filter the word list based on the specified language
+    if language:
+        word_list = [word for word in word_list if word.get('language') == language]
```

Comment:
I would use operator "[]" instead of .get()

Reply:
fixed in https://github.com/SWel1a/QuizBot/pull/11/commits/2e5119bc1c05530bb7bc111e46d81e08dfba42e6